analysis/codeAnalyzer.go

```diff
@@ -1,38 +1,41 @@
 package analysis
 
 import (
-	"github.com/sandstorm/dependency-analysis/dataStructures"
-	"github.com/sandstorm/dependency-analysis/parsing"
 	"os"
 	"path/filepath"
 	"regexp"
+
+	"github.com/sandstorm/dependency-analysis/dataStructures"
+	"github.com/sandstorm/dependency-analysis/parsing"
 )
 
 // mapping from file path to source-unit
-type sourceUnitByFile = map[string][]string
+type sourceUnitByFileType = map[string][][]string
```

Comment:
```suggestion
type sourceUnitsByFileType = map[string][][]string
```

and do something with the `[][]string`